src/include/index/ivf_pq_group.h

```diff
@@ -39,17 +39,15 @@
 #include "index/index_group.h"
 #include "index/ivf_pq_metadata.h"
 
-// flat_ivf_centroids, pq_ivf_centroids,
-// partitioned_pq_vectors, partitioned_pq_index, partitioned_pq_ids
-// cluster_centroids, distance_tables,
+// flat_ivf_centroids, partitioned_pq_vectors, partitioned_pq_index,
+// partitioned_pq_ids, cluster_centroids, distance_tables.
 
 [[maybe_unused]] static StorageFormat ivf_pq_storage_formats = {
     {"0.3",
      {
          // @todo Should these be kept consistent with ivf_flat?
          {"cluster_centroids_array_name", "pq_cluster_centroids"},
          {"flat_ivf_centroids_array_name", "uncompressed_centroids"},
-         {"pq_ivf_centroids_array_name", "partition_centroids"},
```

Comment:
We could also remove the distance tables as they are not used in the new query algorithm.

Reply:
Thanks, removed. Nice to simplify even more.